its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/CodeCoverageTest.java

```diff
@@ -123,6 +130,89 @@ void dotCover_CoverageDirectoryIsNotImported() throws Exception {
       .containsOnly("csharpsquid:S2699");
   }
 
+  private static final String EXCLUSIONS_STRING = "**/Excluded.js";
```

Comment:
I would declare this at the top next to the other constants.

Reply:
Or, maybe better, inline the string to be consistent with the other arguments.

Reply:
I used the constant because I had one grammar error on an argument in the matrix. I can inline it, no problem 👍